lib/blocks.php

```diff
@@ -249,7 +249,7 @@ function do_blocks( $content ) {
 
 	return $rendered_content;
 }
-add_filter( 'the_content', 'do_blocks', 9 ); // BEFORE do_shortcode().
+add_filter( 'the_content', 'do_blocks', 7 ); // BEFORE do_shortcode() and oembed.
```

Comment:
Allows embed blocks to be used as reusable blocks

Reply:
I think this change broke the dynamic blocks in the frontend if you have any meta box in the editor.

Reply:
@youknowriad this change has even more consequences. For one, it completely invalidates `gutenberg_wpautop()` who will have `has_blocks()` always fail (because `do_blocks()` already killed markup), rendering `gutenberg_wpautop()` useless.

Reply:
yes, I noticed too. We're working on fixes in #11050